secure-session/src/main/java/com/solace/samples/cloudfoundry/securesession/controller/SolaceController.java

```diff
@@ -152,18 +153,31 @@ public void init() {
         }
 
         logger.info("Solace client initializing and using SolaceMessagingInfo: " + solaceMessagingServiceInfo);
+        
+        String host = solaceMessagingServiceInfo.getSmfTlsHost();
 
         final JCSMPProperties properties = new JCSMPProperties();
-        properties.setProperty(JCSMPProperties.HOST, solaceMessagingServiceInfo.getSmfTlsHost());
+        properties.setProperty(JCSMPProperties.HOST, host);
         properties.setProperty(JCSMPProperties.VPN_NAME, solaceMessagingServiceInfo.getMsgVpnName());
         properties.setProperty(JCSMPProperties.USERNAME, solaceMessagingServiceInfo.getClientUsername());
         properties.setProperty(JCSMPProperties.PASSWORD, solaceMessagingServiceInfo.getClientPassword());
-
+        
         properties.setProperty(JCSMPProperties.SSL_VALIDATE_CERTIFICATE, true);
         properties.setProperty(JCSMPProperties.SSL_VALIDATE_CERTIFICATE_DATE, true);
         properties.setProperty(JCSMPProperties.SSL_TRUST_STORE, TRUST_STORE);
         properties.setProperty(JCSMPProperties.SSL_TRUST_STORE_PASSWORD, TRUST_STORE_PASSWORD);
 
+		// If using High Availability, the host property will be a comma-separated list of two hosts.
+        if (host.contains(",")) {
+        	// Recommended values for High Availability automatic reconnects.
+			JCSMPChannelProperties channelProperties = (JCSMPChannelProperties) properties
+		            .getProperty(JCSMPProperties.CLIENT_CHANNEL_PROPERTIES);
+			channelProperties.setConnectRetries(1);
+			channelProperties.setReconnectRetries(5);
+			channelProperties.setReconnectRetryWaitInMillis(3000);
+			channelProperties.setConnectRetriesPerHost(20);
+        }
+
         try {
```

Comment:
You have a formatting issue with tabs and spaces. Can we convert all tabs to spaces.